rg_instructor_analytics/views/Suggestion.py

```diff
@@ -117,13 +117,14 @@ def get_percent(total, put):
             return .0 if not (total and put) else float(put) / float(total)
 
         subsections_percent = np.array([get_percent(unit['student_count_in'], unit['student_count']) for unit in units])
+        subsections_percent = subsections_percent[subsections_percent < 1.0]
```

Comment:
Do we really need to take only one of the first two elements from the `subsections_percent` array?

Reply:
numpy's magic! )))